pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -850,6 +851,29 @@ protected CompletableFuture<OffloadPoliciesImpl> internalGetOffloadPolicies(bool
         return getTopicPoliciesAsyncWithRetry(topicName, isGlobal)
             .thenCompose(op -> {
                 TopicPolicies topicPolicies = op.orElseGet(TopicPolicies::new);
+                OffloadPoliciesImpl currentOffloadPolicies = topicPolicies.getOffloadPolicies();
+                if (currentOffloadPolicies != null && offloadPolicies != null){
+                    if (Objects.equals(offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis(),
+                            OffloadPoliciesImpl.DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS)) {
+                        offloadPolicies.setManagedLedgerOffloadDeletionLagInMillis(
+                                currentOffloadPolicies.getManagedLedgerOffloadDeletionLagInMillis());
```

Comment:
I think this will change the behavior. After this change, users are not able to set the lagInMillis to null?